LICENSE.AGPL

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Thanks a lot for your lengthy reply! 

>nor does this prevent other organizations from taking DEDIS code and building on it, modifying it, and using it more-or-less in whatever ways they want. That's what the open-source license is designed to allow. 
[...]
> Does that help?

Yes, that clarifies a lot! The text actually sounded a little different to me with this regard. So I'm definitely OK with this changes now. 

I'm still a little concerned about the static nature of the protocols supported by the current cothority code and the vision that it's possible for node to receive a message like "hey, do you want to run this protocol/validation function on this particular type of data, with this set of particular nodes" - "OK, never heard of this protocol, but sure, why not". This currently isn't possible without going through this repo, DEDIS and the aforementioned CLA and recompiling/restarting a running node. 

But maybe this concern is orthogonal  to the licensing part (?) and we should just make it possible to run different 'protocols' without modifying the code? Would be happy to discuss this further with you soon.

Thanks again for your clarifications.